Packs/AggregatedScripts/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,14 @@
+
+#### Scripts
+
+##### New: BlockUserAppAccessWithCAPolicy
+
+- New: Added a new script- BlockUserAppAccessWithCAPolicy that Checks if a CA policy exists. If yes, adds the user if needed. If no, creates a new CA policy to block access to a specific app for a user.
+- 
+##### New: BlockExternalIPWithCAPolicy
+
+- New: Added a new script- BlockExternalIPWithCAPolicy that Blocks an external IP via Azure Conditional Access using named IP location and policy JSON.
+
+##### New: AddIPToOktaBlockedZone
+
+- New: Added a new script- AddIPToOktaBlockedZone that Checks if an IP is already covered by CIDR/range in Okta BlockedIpZone. If not, adds it.
```

Comment:
New: Added the AddIPToOktaBlockedZone script that checks if an IP is already covered by CIDR/range in Okta BlockedIpZone. If not, it adds it.